modules/dynamic-provisioning-change-default-class.adoc

```diff
@@ -27,20 +34,26 @@ NAME                 TYPE
 gp3 (default)        kubernetes.io/aws-ebs <1>
 standard             kubernetes.io/aws-ebs
 ----
-<1> `(default)` denotes the default storage class.
+<1> `(default)` indicates the default storage class.
 
-. Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` for the default storage class:
+. Make the desired storage class the default by setting the `storageclass.kubernetes.io/is-default-class` annotation to `true`:
 +
 [source,terminal]
 ----
-$ oc patch storageclass gp3 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'
+$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
 ----
++
+[NOTE]
+====
+It is acceptable to have multiple default storage classes for a short time. However, you should ensure that eventually only one default storage class exists.
```

Comment:
I would highlight what happens in that case (the most recently created SC is used)

Reply:
Yeah, according to latest sc time stamp.

Reply:
@lpettyjo remember to add xref to multi/no sc default sc module in note.

Reply:
Done.